_posts/2022-10-27-kani-bolero.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: post
+title: "From Fuzzing to Proof: Using Kani with the Bolero Property-Testing Framework"
+---
+
+Today we're going to talk about how you can use fuzzing and verification in a unified framework, which is enabled by the integration of the [Kani Rust Verifier](https://model-checking.github.io/kani/) in [Bolero](https://camshaft.github.io/bolero/).
+Bolero is a property-testing framework that makes it easy for users to test a piece of Rust code with multiple fuzzing engines, and Kani is a verification tool that allows users to prove certain properties about their Rust code.
+By integrating Kani in Bolero, we enable users to apply multiple verification methods, namely fuzzing and proofs, to the same harness.
```

Comment:
Fuzzing is not verification, so I'd use
```suggestion
By integrating Kani in Bolero, we enable users to apply multiple testing methods, namely fuzzing and verification, to the same harness.
```
But you might argue that verification is not testing either.

Reply:
Done